tasks/main.yml

```diff
@@ -24,6 +24,7 @@
     owner: root
     group: root
     mode: 0644
+    backup: true
```

Comment:
Can you explain ?

Reply:
Making backups of newly provisioned files is the way to go. Again not related to PR, but useful addition imo.